src/CommentClient.js

```diff
@@ -1,51 +0,0 @@
-export default class CommentClient {
```

Comment:
* Moved all *Client classes into `clients/` folder
* Moved all fetch() calls from the components into the clients.
* Created a base client that provides the default fetch config to avoid copy-pasting (eg. cors, cache, headers, token).